armbian/base/scripts/lightningd-bootstrap.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/sh
+# Simple script to bootstrap a running mainnet node.
+
+# Original script
+# https://raw.githubusercontent.com/ElementsProject/lightning/master/contrib/bootstrap-node.sh
+# --------------------------------------------------------------------------------------------
+# Adapted script, by Shift Cryptosecurity AG, Switzerland
```

Comment:
What are the changes from the original?

Reply:
I took out some of the `cli` detection logic and provide the `lightningd` directory. Also, we'll need to keep it somewhat up to date for now, changing the node addresses, until we have a better solution.